src/pseudopeople/schema_entities.py

```diff
@@ -552,17 +717,47 @@ class __Datasets(NamedTuple):
         ),
         date_format=DATEFORMATS.MM_DD_YYYY,
     )
-    # tax_1040: Dataset = Dataset(
-    #     DatasetNames.TAXES_1040,
-    #     columns=list(),
-    #     date_column_name=COLUMNS.tax_year.name,
-    #     state_column_name=COLUMNS.mailing_state.name,
-    #     row_noise_types=(
-    #         NOISE_TYPES.omit_row,
-    #         # NOISE_TYPES.duplication,
-    #     ),
-    #     date_format=DATEFORMATS.MM_DD_YYYY,
-    # )
+    tax_1040: Dataset = Dataset(
+        DatasetNames.TAXES_1040,
+        columns=(  # This defines the output column order
+            COLUMNS.simulant_id,
+            COLUMNS.household_id,
+            COLUMNS.first_name,
+            COLUMNS.middle_initial,
+            COLUMNS.last_name,
+            COLUMNS.mailing_street_number,
+            COLUMNS.mailing_street_name,
+            COLUMNS.mailing_unit_number,
+            COLUMNS.mailing_po_box,
+            COLUMNS.mailing_city,
+            COLUMNS.mailing_state,
+            COLUMNS.mailing_zipcode,
+            COLUMNS.ssn,
+            COLUMNS.spouse_first_name,
+            COLUMNS.spouse_middle_initial,
+            COLUMNS.spouse_last_name,
+            COLUMNS.spouse_ssn,
+            COLUMNS.dependent_1_first_name,
+            COLUMNS.dependent_1_last_name,
+            COLUMNS.dependent_1_ssn,
+            COLUMNS.dependent_2_first_name,
+            COLUMNS.dependent_2_last_name,
```

Comment:
The docs request "type of GQ" columns as well. Do we need household_type here?

Reply:
It says for noise functions only. I can verify with RT. Im also not sure about which row types we need since that is not 100% clear to me in the docs.